config/locales/admin.en.yml

```diff
@@ -23,6 +23,10 @@ en:
         title: 'New channel'
       edit:
         title: 'Edit channel'
+      messages:
+        recurring:
+          error:
+            update: 'Cannot update the recurrence for a channel that has projects associated.'
```

Comment:
se não me engano, seria o contrário: `'Cannot update the recurrence for a channel that has associated projects.'`